src/components/sections/section-wrapper.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react'
+import { CircularLoader } from '@dhis2/ui'
+import UnsavedChangesAlert from '../unsaved-changes-alert'
+
+const SectionWrapper = ({ loading, unsavedChanges, ...props }) => {
```

Comment:
Props only needs to contain `children` here, so `children` should be explicitly listed as a prop.  This also needs a PropTypes declaration